docs/Style-Guide.md

```diff
@@ -67,48 +67,91 @@ Make sure you evade copying other people's work, keep it as original as possible
 
 <br>
 
-## Words to use and avoid
+## Word Choice
 
 <br>
 
 <!-- content here -->
 
-There are a few things you need to use and some to also avoid when writing your documentation. This will improve readbility to your audience and also make your documentation neat, direct and clean all through your writing process.
+There are a few things you need to use and avoid when writing your documentation which will improve readability to your reader and also make your documentation neat, direct and clean all through your writing process.
 
 
 <br>
 
-### Use "you" to address readers
+### Use "you" instead of we or they
 
 
 <br>
 
 <!-- content here -->
 
-Using the word "you" makes the reader know you are speaking to them. This makes it easier to give them direct instruction on what to do when a particular task is explained. 
+Using the word "you" makes the reader know you are speaking to them. It is easier to give them direct instruction on what to do when a particular topic. 
 
- > According to [Wikipidia](#), *You* is usually a secind person pronoun. it is also used to refer to an indeterminate person, as a more common alternative to very formal idefinite pronoun.
+ > According to [Wikipedia](#), ***You*** is usually a second person pronoun. Also, used to refer to an indeterminate person, as a more common alternative to a very formal indefinite pronoun.
```

Comment:
I'd really like to avoid any informal tone in the documentation. Such language is acceptable in "blog posts" but not in reference material. IMHO, a great example of the style our documentation should follow is the Decorators doc -- https://github.com/fastify/fastify/blob/df9fbc183fa05fa1a23781b3f11fbf81f6854033/docs/Decorators.md

I re-worked that whole document to remove "you" language and I think it is much clearer for it.

Reply:
Thank you for the review. 

Will remodel the guide to follow a more formal approach for the documentation. It will be useful to point that out in the documentation as well.

Reply:
@fastify/collaborators might have other opinions.

Reply:
A couple of notes:

1. `we` or `they` does not read well and It's very academic. 
2. The passive voice makes concepts harder to learn.

As an example, I think it's totally ok in using `you` in https://github.com/fastify/fastify/blob/df9fbc183fa05fa1a23781b3f11fbf81f6854033/docs/Plugins-Guide.md. However that piece is not as crisp as https://github.com/fastify/fastify/blob/df9fbc183fa05fa1a23781b3f11fbf81f6854033/docs/Decorators.md is. I do not see using a more formal language in contrast with using `you` on a guide.

Note that we might want to differentiate between guides and references material.

Reply:
> Note that we might want to differentiate between guides and references material.

Agreed. And our documentation needs to be clearly delineated between the two. See https://nodejs.org/en/docs/. The actual API docs, e.g. `http.IncomingMessage`, are void of any human references. But the "Guides" are informal and conversational. Guides are good for teaching, and reference is good for... referencing (acquiring a deeper knowledge of the actual implementations).

Reply:
> Note that we might want to differentiate between guides and references material.

I agree with this as well. The documentation will need to point out a clear difference between the two.